command_line_assistant/commands/chat.py

```diff
@@ -534,7 +534,11 @@ def _validate_query(self):
             ChatCommandException: In case the query has invalid sizing (less than 1 character).
         """
         # If both are empty, raise exception
-        if not self.args.query_string and not self.args.stdin:
+        if (
+            not self.args.query_string
+            and not self.args.stdin
+            and not self.args.with_output
+        ):
             logger.debug("Both query_string and stdin are empty.")
             raise ChatCommandException(
                 "Your query needs to have at least 2 characters. Either query or stdin are empty."
```

Comment:
Do we want to adjust the error message to reflect this change?

Reply:
the "if both are empty" comment above needs updating too

Reply:
good point. Let me update

Reply:
I removed that condition as it is not necessarily required anymore. 
We fixed this case under https://github.com/rhel-lightspeed/command-line-assistant/pull/383